static/templates/actors/familiar-sheet.hbs

```diff
@@ -111,7 +111,7 @@
                             </div>
                             <div class="rollable skills-attack" title="{{data.attack.breakdown}}" data-attack-roll="">
                                 <strong>{{localize "PF2E.Familiar.AttackRoll"}}</strong>
-                                <span>{{numberFormat data.attack.value decimals=0 sign=true}}</span>
+                                <span>{{numberFormat data.attack.check.mod decimals=0 sign=true}}</span>
```

Comment:
If I recall, we can do data.attack.mod directly correct? Its dc that returns an object.